src/Pages/MembershipApplication/MembershipPlan.js

```diff
@@ -18,14 +19,20 @@ class MembershipPlan extends Component {
       img1: winter,
       img2: winter2,
       activeId: undefined,
-      planSelected: false
+      planSelected: false,
+      yearPlan: getYearPlan(),
+      year: new Date().getFullYear(),
+      semester: getSemesterPlan()
     }
   }
 
   cardSelected (id) {
```

Comment:
give this a more descriptive parameter name. imagine we are a freshman and it's our first time looking at a big project like this.

```suggestion
   cardSelected (cardTitle) {
```